utils/funcexp/functor.h

```diff
@@ -82,6 +84,14 @@ class Func
     throw logging::IDBExcept(oss.str(), logging::ERR_DATATYPE_NOT_SUPPORT);
   }
 
+  bool isCompilable(const execplan::CalpontSystemCatalog::ColType& colType)
+  {
+    return false;
+  }
+  llvm::Value* compile()
+  {
+  }
```

Comment:
It looks like the actual interesting part is still missing. The body of `compile()` seems to be missing `return nullptr;`.